FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -12,6 +14,25 @@ internal class ExecutionTest : IntegrationTest
         private static string SYSTEM_CONNECTION_STRING = $"database={Database};clientid={ClientId};clientsecret={ClientSecret};endpoint={Endpoint};account={Account};env={Env}";
         private static string USER_CONNECTION_STRING = $"{SYSTEM_CONNECTION_STRING};engine={EngineName}";
         private static string SYSTEM_WRONG_CREDENTIALS = $"database={Database};clientid={ClientId};clientsecret=wrongClientSecret;endpoint={Endpoint};account={Account};env={Env}";
+        private static string CREATE_TABLE = @"
+            CREATE FACT TABLE IF NOT EXISTS ALL_TYPES
+            (
+                i INTEGER  NULL,
+                n NUMERIC NULL,
+                bi BIGINT NULL,
+                r REAL NULL,
+                dp DOUBLE PRECISION NULL,
+                t TEXT NULL,
+                d DATE NULL,
+                ts TIMESTAMP NULL,
+                tstz TIMESTAMPTZ NULL,
+                b BOOLEAN NULL
```

Comment:
It's missing `BYTEA`, `ARRAY` and `DECIMAL` types

Reply:
Yes, I have opened separate issue for arrays. I will add decimal however

Reply:
Decimal is added

Reply:
`BYTEA`?

Reply:
I have created issue FIR-27494 for all arrays including bytea and starting working on it in separate branch.